serialization/src/main/scala/sbt/serialization/json/JSONPickleFormat.scala

```diff
@@ -0,0 +1,574 @@
+package sbt.serialization
+
+import scala.pickling.{
+  FastTypeTag,
+  Output,
+  PBuilder,
+  PReader,
+  Pickle,
+  PickleFormat,
+  PickleTools,
+  PicklingException,
+  SPickler,
+  StringOutput,
+  Unpickler,
+  UnpickleOps
+}
+import scala.pickling.internal.lookupUnpicklee
+// FIXME this isn't threadsafe right? we need to get rid of its use.
```

Comment:
wasn't this about reflection? but now it appears to be about json4s import right below it